File: openapi/code_samples/python/utilitypayments/get_all_billers/code.py
import requests

url = "https://utilities.reloadly.com/billers"

payload={}
headers = {
  'Authorization': 'Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
  'Accept': 'application/com.reloadly.utilities-v1+json',
}

response = requests.request("GET", url, headers=headers, data=payload)

print(response.text)

File: openapi/code_samples/nodejs/utilitypayments/get_all_billers/code.js
var request = require('request');
var options = {
  'method': 'GET',
  'url': 'https://utilities.reloadly.com/billers',
  'headers': {
    'Authorization': 'Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
    'Accept': 'application/com.reloadly.utilities-v1+json',
  }
};
request(options, function (error, response) {
  if (error) throw new Error(error);
  console.log(response.body);
});